R/estimate_vaccineff.R

```diff
@@ -122,20 +122,40 @@ summary.vaccineff <- function(object, ...) {
   stopifnot("Input must be an object of class 'vaccineff'" =
       checkmate::test_class(object, "vaccineff")
```

Comment:
These lines shouldn't be required as `summary()` is a S3 generic and this function is a S3 method dispatching when a `<vaccineff>` object is supplied to `object` there shouldn't be any cases where this function is run and the `object` is not a `<vaccineff>` object. 

Let me know if I've misunderstood this.

Reply:
You're right, they are unnecessary. Solved for both functions in 062d6a2b55cd2dc08d2c435871cab32d6c9144d1 and f1893fa4ee5d94be03c448ced898f95d6711854d